File: thegamer.com/_integrations/C#/HttpClient.cs
var client = new HttpClient();
var request = new HttpRequestMessage
{
	Method = HttpMethod.Get,
	RequestUri = new Uri("https://apitube.io/v1/sets/67017857?limit=250&offset=0"),
	Headers =
	{
		{ "X-ApiTube-Key", "YOUR-KEY-HERE" },
	},
};
using (var response = await client.SendAsync(request))
{
	response.EnsureSuccessStatusCode();
	var body = await response.Content.ReadAsStringAsync();
	Console.WriteLine(body);
}

File: thegamer.com/_integrations/Python/http_client.py
import http.client

conn = http.client.HTTPSConnection("apitube.io")

headers = {
    'X-ApiTube-Key': "YOUR-KEY-HERE"
}

conn.request("GET", "/v1/sets/67017857?limit=250&offset=0", headers=headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))
